.github/release.yml

```diff
@@ -4,7 +4,14 @@
 changelog:
   exclude:
     labels:
+      - duplicate
+      - invalid
+      - modulesync
```

Comment:
I don't think we use the modulesync label in this project?

Reply:
Copy and past error 😅